src/main/java/com/lotto/game/Game.java

```diff
@@ -0,0 +1,46 @@
+package com.lotto.game;
+
+import com.lotto.model.LottoResult;
+import com.lotto.model.Lottos;
+import com.lotto.model.WinningNumbers;
+import com.lotto.type.RewardType;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Game {
+    private static final int LOTTO_PRICE = 1000;
+
+    //구입하다.
+    public static Lottos purchase(int money) {
+        return new Lottos(money / LOTTO_PRICE);
+    }
+
+    //당첨번호 입력받다.
+    public static WinningNumbers makeWinningNumbers(List<Integer> numbers) {
+        Set<Integer> numberSet = new HashSet<>();
+
+        numbers.stream().forEach((number) -> {
```

Comment:
중복체크 좋습니다 👍 💯 
단순 반복이시라면 굳이 `stream`은 생성 안하셔도 될 것 같습니다 😄 

```suggestion
        numbers.forEach((number) -> {
```